modules/mssqlserver/src/test/java/org/testcontainers/containers/MSSQLServerContainerTest.java

```diff
@@ -0,0 +1,46 @@
+package org.testcontainers.containers;
+
+import com.zaxxer.hikari.HikariConfig;
+import com.zaxxer.hikari.HikariDataSource;
+import org.junit.Rule;
+import org.junit.Test;
+
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+
+import static org.rnorth.visibleassertions.VisibleAssertions.assertEquals;
+import static org.rnorth.visibleassertions.VisibleAssertions.fail;
+
+/**
+ * @author Enrico Costanzi
+ */
+public class MSSQLServerContainerTest {
```

Comment:
I think we don't need this test class, since we have it already in the `jdbc-test` module.
@rnorth what was the reason, for having the tests in their own module?

Reply:
Hi @kiview should I remove this test? What do you think about my last comment?

Reply:
Oh you are right, let's move the test there like you have proposed.